tests/queries/0_stateless/01947_multiple_pipe_read.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+SAMPLE_FILE="$CURDIR/01947_sample_data.csv"
+STD_ERROR_CAPTURED="$CURDIR/01947_std_error_captured.log"
+
+echo 'File generated:'
+${CLICKHOUSE_LOCAL} -q "SELECT number, if(number in (4,6), 'AAA', 'BBB') from numbers(7) FORMAT CSV" --format_csv_delimiter=, >"$SAMPLE_FILE"
+cat "$SAMPLE_FILE"
+
+echo '******************'
+echo 'Attempt to read twice from a pipeline'
+${CLICKHOUSE_LOCAL} --structure 'key String' -q 'select * from table; select * from table;' <<<foo
```

Comment:
Also it is better not to use this in tests, since the behavior is different in different versions on bash:

```
$ echo -n "$BASH_VERSION: "; test -p /dev/fd/0 <<<foo && echo pipe || echo regular
5.1.8(1)-release: pipe
$ docker run --rm -it ubuntu:20.04 bash -c 'echo -n "$BASH_VERSION: "; test -p /dev/fd/0 <<<foo && echo pipe || echo regular'
5.0.17(1)-release: regular
```

You already have test with pipe (`echo 1 | clickhouse-local`), this is enough.

Reply:
Removed that test